scene/resources/2d/tile_set.cpp

```diff
@@ -4931,10 +4931,13 @@ void TileSetAtlasSource::_get_property_list(List<PropertyInfo> *p_list) const {
 		}
 
 		for (const KeyValue<int, TileData *> &E_alternative : E_tile.value.alternatives) {
+			const String formatted_key = vformat("%d", E_alternative.key);
```

Comment:
```suggestion
			const String formatted_key = itos(E_alternative.key);
```

Since it's simply `int` to `String`.